functions.sh

```diff
@@ -336,6 +336,19 @@ nameservers() {
   dig +short ns "${1}" | sort | tr "[:upper:]" "[:lower:]"
 }
 
+function realpath(){
```

Comment:
Greg's Bash Wiki [recommends](https://mywiki.wooledge.org/BashPitfalls#function_foo.28.29) that the `function` keyword not be combined with parenthesis. Please use either:

```suggestion
function realpath {
```

or 

```suggestion
realpath() {
```